src/test/java/hudson/plugins/ec2/EC2CloudTest.java

```diff
@@ -27,6 +27,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class EC2CloudTest {
 
+    @Test
+    public void testFileBasedSShKey() {
+        assertTrue("file content should not have been empty", EC2Cloud.fetchPrivateKeyFromDisk(getClass().getClassLoader().getResource("hudson/plugins/ec2/test.pem").getPath()) != null);
```

Comment:
please, use `assertNull` or `assertNotNull` if that's what you want to check. If the test fails for whatever reason, although we get the `file content should not have been empty` message, the reason of the failure would be `expected true but was false. `assertTrue` and `assertFalse` should be used only to assert boolean values